app/models/smart_proxy.rb

```diff
@@ -90,6 +90,25 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
+  def version
+    begin
+      version = { :status => N_('OK'), :message => ProxyAPI::Version.new(:url => url).version }
+    rescue => e
+      version = { :status => N_('Error'), :message => _('Unable to communicate with the proxy: %s') % e }
```

Comment:
Perhaps disregard the translation comment, it might be better to use a "success" boolean as the LDAP test does?